wandb/sdk/wandb_settings.py

```diff
@@ -1851,7 +1856,20 @@ def _apply_init(self, init_settings: Dict[str, Union[str, int, None]]) -> None:
 
         # update settings
         self.update(init_settings, source=Source.INIT)
+        self._handle_rewind_logic()
+        self._handle_resume_logic()
 
+    def _handle_rewind_logic(self) -> None:
+        # handle rewind logic
```

Comment:
this comment doesn't seem to serve a purpose; it's just the function name?

Reply:
fixed